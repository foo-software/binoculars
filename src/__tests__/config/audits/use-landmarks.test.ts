import auditUseLandmarks from '../../../config/audits/use-landmarks';

describe('use-landmarks audit', () => {
  it('should match snapshot', () => {
    expect(auditUseLandmarks('en')).toMatchSnapshot();
  });
});
