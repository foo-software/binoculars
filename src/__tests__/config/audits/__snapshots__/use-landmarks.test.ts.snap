// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`use-landmarks audit should match snapshot 1`] = `
Object {
  "description": "Landmark elements (<main>, <nav>, etc.) are used to improve the keyboard navigation of the page for assistive technology. [Learn more](https://web.dev/use-landmarks/).",
  "title": "HTML5 landmark elements are used to improve navigation",
}
`;
